src/hooks/useNetworkMonitor.ts

```diff
@@ -18,7 +18,7 @@ export type NetworkRequest = {
     headers: Header[]
     body: {
       query: string
-      variables?: object
+      variables?: Record<string, unknown>
```

Comment:
I changed the types to `Record<string, unknown>` to enable the narrowing on the `in` operator